### ISSUE TITLE
Views syncing on every migrate

### ISSUE THREAD

**Issue by kochhar** [mypebble/django-pgviews#38]
I have created some materialized views using `view.MaterializedView` as a base class. I find that every time I run migration, the views are recreated -- dropped and created again. 

Is there some way to disable this behaviour?

**Comment by scott-w** [comment]
Thanks for posting this. If I recall correctly, this is done so we can migrate the underlying tables without Postgres raising an error when a view depends on a field you may be removing. If you have a potential resolution then I’d be more than happy to look over a PR and ask @tpict to prepare a release.

**Comment by kochhar** [comment]
Good point, @scott-w. I'm not very familiar with how migrations are implemented. 

Do you think it would be possible to examine the migration to determine if the underlying table has been updated? If so, I could create a patch for this.

**Comment by petrprikryl** [comment]
Hello, maybe you can inspire here https://pypi.org/project/django-migrate-sql-deux/. It generates non-standard Django migrations for own SQL objects managed in Python. Very similar to django-pgviews. I think this technique comes very handy on large project with many views and other SQL objects.
